### PR TITLE
EVA-2252 Get database name

### PR DESCRIPTION
**Pull request by andresfsilva**

### CODE REVIEW

File: ebi_eva_common_pyutils/metadata_utils.py [hunk 1]
```diff
@@ -48,3 +48,18 @@ def get_dbsnp_mirror_db_info(pg_metadata_dbname, pg_metadata_user, pg_metadata_h
...
+    query = "select t.taxonomy_code, a.assembly_code " \
+            "from assembly a " \
+            "left join taxonomy t on (t.taxonomy_id = a.taxonomy_id)" \
+            "where assembly_accession = '{0}'".format(assembly)
+    rows = get_all_results_for_query(metadata_connection_handle, query)
+    if len(rows) == 0:
+        raise ValueError(f'No database for {assembly} found')
+    elif len(rows) > 1:
+        options = ', '.join((f'{r[0]}_{r[1]}' for r in rows))
+        raise ValueError(f'More than one possible database for {assembly} found: {options}')
```
Comment (tcezard) [inline ebi_eva_common_pyutils/metadata_utils.py:63]: I don't think that we should raise if there is no database just return `None`. 
I guess we don't expect to have more than one db name per assembly so raising when more than one makes sense.

**Review by tcezard — CHANGES_REQUESTED** [review]

File: ebi_eva_common_pyutils/metadata_utils.py [hunk 2]
```diff
@@ -48,3 +48,18 @@ def get_dbsnp_mirror_db_info(pg_metadata_dbname, pg_metadata_user, pg_metadata_h
         dbsnp_mirror_db_info = [{"dbsnp_build": result[0], "pg_host": result[1], "pg_port": result[2]}
                                 for result in get_all_results_for_query(pg_conn, dbsnp_mirror_db_info_query)]
     return dbsnp_mirror_db_info
+
+
+def get_variant_warehouse_db_name_from_assembly(metadata_connection_handle, assembly):
```
Comment (apriltuesday) [inline ebi_eva_common_pyutils/metadata_utils.py:53]: in the submission automation, i check the taxonomy id in addition to the assembly accession when getting the db name... is that something we should do here as well?

File: ebi_eva_common_pyutils/metadata_utils.py (hunk 1 above)
Comment (andresfsilva) [reply ebi_eva_common_pyutils/metadata_utils.py]: Agree, changed to return None instead

File: ebi_eva_common_pyutils/metadata_utils.py (hunk 2 above)
Comment (andresfsilva) [reply ebi_eva_common_pyutils/metadata_utils.py]: Yes, I used the taxonomy in the count scripts and updated the method to request for the taxonomy as well

**Review by apriltuesday — APPROVED** [review]

**Review by tcezard — APPROVED** [review]

**Pull request merged**